playground/docker-compose.fork.yml

```diff
@@ -163,5 +171,31 @@ services:
         - ETH_RPC_URL=http://127.0.0.1:8545
     ports:
       - 8001:80
+
+  chain-check:
```

Comment:
This whole section seems like it should rather be part of the chain directly.
Actually I'm a bit surprised that apparently it's possible to set the allow list code before anvil can return a block. But maybe the easiest solution would be to have a readiness check in the form of:
```
if no_current_block {
    return false;
}
anvil_setCode()
```

Reply:
Yeah, true. A separate service was the fastest option to test it. Replaced with an inner health check.